pint_server/pint_db_migrate/versions/b720d8b8b775_set_server_type_name_not_null.py

```diff
@@ -0,0 +1,62 @@
+"""Set server type, name NOT NULL
+
+Revision ID: b720d8b8b775
+Revises: b381843009e0
+Create Date: 2021-09-06 17:31:28.809521
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'b720d8b8b775'
+down_revision = 'b381843009e0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('amazonservers', 'type',
+               existing_type=postgresql.ENUM('region', 'update', name='server_type'),
+               nullable=False)
+    op.alter_column('amazonservers', 'name',
+               existing_type=sa.VARCHAR(length=100),
+               nullable=False)
+    op.alter_column('googleservers', 'type',
+               existing_type=postgresql.ENUM('region', 'update', name='server_type'),
+               nullable=False)
+    op.alter_column('googleservers', 'name',
+               existing_type=sa.VARCHAR(length=100),
+               nullable=False)
+    op.alter_column('microsoftservers', 'type',
+               existing_type=postgresql.ENUM('region', 'update', name='server_type'),
+               nullable=False)
+    op.alter_column('microsoftservers', 'name',
+               existing_type=sa.VARCHAR(length=100),
+               nullable=False)
+    # ### end Alembic commands ###
+
+
+def downgrade():
```

Comment:
Same as above; we can define the ENUM once, and reference it in all the statements.